packages/app-store/routing-forms/pages/forms/[...appPages].tsx

```diff
@@ -342,22 +342,12 @@ export default function RoutingForms({
   );
 }
 
-const ShellContainer = ({ page }: { page: React.ReactElement }) => {
-  const { t } = useLocale();
-
+RoutingForms.getLayout = ({ children }: { children: React.ReactNode }) => {
   return (
-    <Shell
-      title={t("routing_forms")}
-      description={t("routing_forms_description")}
```

Comment:
not needed because they're for SEO purposes and we handle SEO natively in app router